core/src/components/input/test/shape/input.e2e.ts

```diff
@@ -0,0 +1,129 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * Fill is only available in MD mode
```

Comment:
Should this comment reference shape? Maybe we can say that the fill is only supported by the `md` theme and `ionic` theme so it doesn't make sense to test shape in the other themes?

Reply:
I'm not sure I follow. We should be testing the shape with the solid fill to prevent visual regression. Right now, `md` is the only theme that supports the solid fill.